prodserverdock/Dockerfile

```diff
@@ -1,24 +0,0 @@
-FROM phusion/baseimage
```

Comment:
this is not part of StakeManager. 
I understand your urge to clean the repo, and it can be done - in a separate commit.